Fitter/scripts/EFTFitter.py

```diff
@@ -405,12 +432,18 @@ def gridScan(self, name='.test', batch='', freeze=False, scan_params=['ctW','ctZ
         ### Runs deltaNLL Scan in two parameters using CRAB or Condor ###
         logging.info("Doing grid scan...")
 
-        CMSSW_BASE = os.getenv('CMSSW_BASE')
-        if not "/afs/" in workspace:
+        CMSSW_BASE = self.__override_CMSSW_BASE()
+
+        print(f"CMSSW_BASE after manipulation is {CMSSW_BASE}")
+
+        if not (workspace.startswith("/afs/") or workspace.startswith("/users/") or  workspace.startswith("/scratch365/")):
```

Comment:
Should we just check for a leading `/` instead. I know this covers the basis on glados and lxplus, but if a file system doesn't use one of these three paths it could still cause issues.